iosHyperskillApp/iosHyperskillApp/Sources/Modules/FirstProblemOnboarding/Views/FirstProblemOnboardingContentView.swift

```diff
@@ -0,0 +1,128 @@
+import SwiftUI
+
+extension FirstProblemOnboardingContentView {
+    struct Appearance {
+        let interitemSpacing = LayoutInsets.smallInset
+
+        let illustrationHeight: CGFloat = 380
+
+        let maxWidth: CGFloat = DeviceInfo.current.isPad ? 400 : .infinity
+    }
+}
+
+struct FirstProblemOnboardingContentView: View {
+    private(set) var appearance = Appearance()
+
+    @Environment(\.horizontalSizeClass) private var horizontalSizeClass
+
+    private let actionButtonsFeedbackGenerator = FeedbackGenerator(feedbackType: .selection)
+
+    let title: String
+
+    let subtitle: String
+
+    let buttonText: String
+
+    let isNewUserMode: Bool
+
+    let onCallToActionButtonTap: () -> Void
+
+    var body: some View {
+        VerticalCenteredScrollView(showsIndicators: false) {
+            VStack(spacing: 0) {
+                if horizontalSizeClass == .regular {
+                    Spacer()
```

Comment:
Thank you for this explanation! I think that if we will have the third onboarding screen with the same layout, we will create abstraction to avoid code duplication